docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena provides capability to manage persistent data using volumes.
+
+Volumes are supported as first-class citizens in both Kontena Master API and on stack yaml definitions.
+
+Currently Kontena does not automatically configure any volume drivers within the grid so user must configure those themselves.
```

Comment:
Kontena tracks the available Docker volume drivers on each host node, as shown in `kontena node show`, and will only deploy services using such a volume driver onto a host node that provides that volume driver. In order to use any other volume drivers than the default `local` driver, those volume drivers must be provided by Docker plugins on the host nodes. Kontena itself does not yet support provisioning Docker plugins onto the grid's  host nodes.